httpmw/Ip_cognito_auth_test.go

```diff
@@ -27,16 +32,28 @@ func (c *cognitoIdentityProviderClientMock) GetUser(input *cognitoidentityprovid
 	return args.Get(0).(*cognitoidentityprovider.GetUserOutput), args.Error(1)
 }
 
+func getJWTtoken() (string, error) {
+	claims := jwt.MapClaims{
+		"client_id": authTestCogntoClientId,
+	}
+
+	jwtToken := jwt.NewWithClaims(jwt.SigningMethodHS256, claims)
+	token, err := jwtToken.SignedString([]byte{})
+
+	return token, err
+}
+
 func TestCognitoIPSucceed(t *testing.T) {
 	assert := assert.New(t)
-	ipRanges := "192.168.10.1-192.168.10.10,192.168.20.1-192.168.20.10"
+	ipRanges := authTestIPRangesLarge
```

Comment:
This variable is redundant you can just pass `authTestIPRangesLarge` directly into  `IpCognitoAuth`.